jsonifier/models/models.py

```diff
@@ -105,6 +105,13 @@ def _jsonify_record(self, parser, rec, root):
                     value, json_key = self._jsonify_record_handle_resolver(
                         rec, field, resolver, json_key
                     )
+            # whatever json value we have found in subparser or not ass a sister key
+            # on the same level {jsoni_key}_fieldname
```

Comment:
It looks like this comment is outdated, might want to review this.

Reply:
fixed